service/src/main/resources/logback.xml

```diff
@@ -45,7 +45,7 @@
         level="ERROR" />
     <logger name="springfox.documentation" level="ERROR" />
 
-    <logger name="com.ge.predix.acs" level="DEBUG" additivity="false">
```

Comment:
Siva,
Maybe we should only disable DEBUG for messages which are printed per policy for now ?  Can we do another test ?  
Especially because we have no way to changing log levels at runtime. It would be good to still have some high level debugging for troubleshooting in production.